modules/reference/pages/rpk/rpk-transform/rpk-transform-pause.adoc

```diff
@@ -0,0 +1,34 @@
+= rpk transform pause
+:description: rpk transform pause
+
+Pause a data transform.
+
+This command suspends execution of the specified transform without removing it from the system. In this way, a transform may resume at a later time, with each new processor picking up processing from the last committed offset on the corresponding input partition.
+
+Subsequent `rpk transform list` operations will show transform processors as `inactive`.
+
+To resume a paused transform, use `rpk transform resume`.
+
+== Usage
+
+[,bash]
+----
+rpk transform pause [NAME] [flags]
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|-h, --help |- |Help for pause.
+
+|--config |string |Redpanda or `rpk` config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Comment:
```suggestion
|--config |string |Redpanda or `rpk` configuration file. Default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Reply:
Rejecting this change for consistency, as we would need to apply it globally. If necessary we can do it in another PR.